i18n/validate.py

```diff
@@ -53,9 +54,9 @@ def msgfmt_check_po_file(locale_dir, filename):
     # Use relative paths to make output less noisy.
     rfile = os.path.relpath(filename, locale_dir)
     out, err = call('msgfmt -c -o /dev/null {}'.format(rfile), working_directory=locale_dir)
-    if err != '':
-        log.info('\n' + out)
-        log.warning('\n' + err)
+    if err:
+        log.info(six.u('\n') + out.decode('utf8'))
+        log.warning(six.u('\n') + err.decode('utf8'))
```

Comment:
UTF-8 is the right thing to use here, Thanks.
One last tweak: there's no need here for six.u, you can just use `u'\n'` since we don't support Python 3 less than 3.3.